src/main/java/uk/gov/pay/adminusers/persistence/entity/InviteEntity.java

```diff
@@ -144,27 +175,25 @@ public Invite toInvite() {
         return new Invite(email, telephoneNumber);
     }
 
-    /**
-     * isExpired()
-     * <p>
-     * Calculates expire based on midnight (hence the truncation to days that sets the day to Midnight)
-     * It takes the date when the Invite was created and truncate it to days (set it to Midnight), adds the EXPIRY_DAYS
-     * calculates if this date result is after Now. IE:
-     * <p>
-     * - Invite created 01-05-2017 10:15:20
-     * - isExpired() checked on 02-05-2017 17:50:10
-     * <p>
-     * - Truncating Invite created -> 01-05-2017 00:00:00
-     * - Being EXPIRY_DAYS = 2, SUM these to the previous days -> 03-05-2017 00:00:00
-     * <p>
-     * - Note: EXPIRY_DAYS = 2 is equivalent to next day Midnight (the following day after next at 00:00:00) so the
-     * Invite will be valid until next day until 23:59:59:999
-     * <p>
-     * - Invite is not expired because 02-05-2017 17:50:10 < 03-05-2017 00:00:00
-     *
-     * @return true if expired
-     */
     public boolean isExpired() {
-        return now(UTC).isAfter(date.truncatedTo(DAYS).plus(EXPIRY_DAYS, DAYS));
+        return now(UTC).isAfter(expiryDate);
+    }
+
+    public UserEntity mapToUserEntity() {
```

Comment:
See comment below. I think it would be better to create a User.java instead of UserEntity here..